src/cloud/ShareAudioToolbar.cpp

```diff
@@ -0,0 +1,245 @@
+/*  SPDX-License-Identifier: GPL-2.0-or-later */
+/*!********************************************************************
+
+  Audacity: A Digital Audio Editor
+
+  ShareAudioToolbar.cpp
```

Comment:
What was the reasoning to put this in `src/cloud` but the rest in `src/cloud/audiocom`?

Reply:
I’m not asking for a change.